spring-batch-docs/asciidoc/jsr-352.adoc

```diff
@@ -204,7 +206,28 @@ To use Spring dependency injection within a JSR-352 based batch job consists of
         </step>
     </job>
 </beans>
-        
+----
+
+[source, java, role="javaContent"]
+----
+@Configuration
+public class BatchConfiguration {
+
+    @Bean
+    public Batchlet fooBatchlet() {
+        FooBatchlet batchlet = new FooBatchlet();
+        batchlet.setProp("bar");
+        return batchlet;
+       }
+}
+
+
+<?xml version="1.0" encoding="UTF-8"?>
+<job id="fooJob" xmlns="http://xmlns.jcp.org/xml/ns/javaee" version="1.0">
+    <step id="step1" >
+        <batchlet ref="fooBatchlet" />
+    </step>
+</job>
```

Comment:
We probably want to have a ifdef block for html with what is in the PR.  Then add  a ifdef block for pdf that will enumerate that they can use either java config or XML to configure the fooBatchlet.   I can do this on merge if you agree.

Reply:
Let me update...